app/database/models.py

```diff
@@ -30,5 +32,36 @@ class Event(Base):
     start = Column(DateTime, nullable=False)
     end = Column(DateTime, nullable=False)
     owner_id = Column(Integer, ForeignKey("users.id"))
+    category_id = Column(Integer, ForeignKey("categories.id"))
 
     owner = relationship("User", back_populates="events")
+
+
+class Category(Base):
+    __tablename__ = "categories"
+
+    __table_args__ = (
+        UniqueConstraint('user_id', 'name', 'color'),
+    )
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String, nullable=False)
+    color = Column(String, nullable=False)
+    user_id = Column(Integer, ForeignKey("users.id"), nullable=False)
+
+    @staticmethod
+    def create(db_session: Session, name: str, color: str, user_id: int):
```

Comment:
Excellent implementation of this class, kudos!

Improvement idea: Use `@classmethod` and put the `cls` instead of `Category`

Reply:
Thank you :)